packages/angular-cli/models/webpack-build-utils.ts

```diff
@@ -109,3 +109,18 @@ export function extraEntryParser(
       return extraEntry;
     });
 }
+
+const hashFormats: { [option: string]: any } = {
+  'none':     { chunk: '',             extract: '',                  file: '[name]'    },
+  'media':    { chunk: '',             extract: '',                  file: '[hash:20]' },
```

Comment:
**hashlength note:** I think a 20 character long hash is unnecessary and make it much more less human readable. 6 would be much better.

**media (file) template format note:** as stated in https://github.com/angular/angular-cli/pull/3871#issuecomment-270905095. `sky-bg.jpg` becomes `8bcfb98e274f484ebac02d81670b7e4c.jpg` with `'[hash:20]'`. Don't you think `[name].[hash:20]` would be more ideal? Looking at `8bcfb98e274f484ebac02d81670b7e4c.jpg` have absolutely no idea which image it is, that is a problem is think.